nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/nifi/dbcp/hive/Hive3ConnectionPool.java

```diff
@@ -178,6 +278,12 @@ protected void init(final ControllerServiceInitializationContext context) {
         props.add(MAX_WAIT_TIME);
         props.add(MAX_TOTAL_CONNECTIONS);
         props.add(VALIDATION_QUERY);
+        props.add(MIN_IDLE);
```

Comment:
These REALLY should have had some unit tests associated with them, especially with the use of Expression Language. None of them appear to actually have EL evaluated, so none of them work with EL. I'll write up a separate Jira for this since NIFI-6684 went into a release.

Reply:
Hey @mattyb149. Your right, my bad.  It's not much of an excuse on my end, but basically I compared this to the previous ticket where properties were added, saw that it was a copy/paste from that previous addition to the DBCP properties, and now we have properties in both that say they allow EL but don't actually.... https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java#L312 Is the same.

Reply:
No worries, I added https://issues.apache.org/jira/browse/NIFI-6954 to cover the fix for DBCPConnectionPool as well, hope to get to that shortly